scripts/sandbox_start.sh

```diff
@@ -15,8 +15,6 @@ setupFabric() {
 # TODO we need health check for peer
 sleep 20
 cd test/bdd && go test
-curl -d '{"name":"vc-issuer-1", "uri":"http://vc-issuer-1.com", "signatureType":"Ed25519Signature2018"}' -H "Content-Type: application/json" -X POST http://localhost:8070/profile
-curl -d '{"name":"vc-issuer-2", "uri":"http://vc-issuer-2.com", "signatureType":"Ed25519Signature2018", "did":"did:v1:test:nym:z6MkhLbRigh9utJNCaiEAdkqktz4r7yVBFDeaeqCeT7pRFnF","didPrivateKey":"5dF8yAW7hjLkJsfMXKqTPdDZUT56dX7Jq7TdXEtUEHHt2YUFAE34nQwyPCEp5XdWCKPSxs69xXqozsNh6MoJTmz5"}' -H "Content-Type: application/json" -X POST http://localhost:8070/profile
```

Comment:
This seemed to be redundant with the `curl` calls that are done in vcs_configure.sh, so I removed them from here.